ipaclient/plugins/vault.py

```diff
@@ -558,74 +557,79 @@ def forward(self, *args, **options):
         return response
 
 
-class _TransportCertCache(collections.MutableMapping):
+class _TransportCertCache(object):
     def __init__(self):
         self._dirname = os.path.join(
-                USER_CACHE_PATH, 'ipa', 'kra-transport-certs')
-        self._transport_certs = {}
+                USER_CACHE_PATH, 'ipa', 'kra-transport-certs'
+        )
 
     def _get_filename(self, domain):
         basename = DNSName(domain).ToASCII() + '.pem'
         return os.path.join(self._dirname, basename)
 
-    def __getitem__(self, domain):
-        try:
-            transport_cert = self._transport_certs[domain]
-        except KeyError:
-            transport_cert = None
+    def load_cert(self, domain):
```

Comment:
Can we rename this to `get`? The `_cert` suffix seems superfluous, given the class is named `_Transport*Cert*Cache`.